aws/internal/service/ec2/waiter/status.go

```diff
@@ -85,12 +85,7 @@ const (
 // ClientVpnAuthorizationRuleStatus fetches the Client VPN authorization rule and its Status
 func ClientVpnAuthorizationRuleStatus(conn *ec2.EC2, authorizationRuleID string) resource.StateRefreshFunc {
 	return func() (interface{}, string, error) {
-		endpointID, targetNetworkCidr, accessGroupID, err := tfec2.ClientVpnAuthorizationRuleParseID(authorizationRuleID)
```

Comment:
These changes for EC2 Client VPN look unrelated and I'm not sure their impact.

Reply:
Heh, yeah. I saw these when I was creating `SecurityGroupStatus()` and `finder. SecurityGroupByID()`. It inlines what already exists in `finder.ClientVpnAuthorizationRuleByID()` maybe shouldn't really be included here.